src/test/java/org/fcrepo/camel/tests/FcrepoLdpathIT.java

```diff
@@ -177,15 +177,13 @@ public void testLdpathService() throws Exception {
         final List<Map<String, List<String>>> data4 = MAPPER.readValue(response4, List.class);
 
         assertFalse(data4.isEmpty());
-        assertTrue(data4.get(0).containsKey("label"));
+        assertTrue(data4.get(0).containsKey("hasParent"));
         assertTrue(data4.get(0).containsKey("type"));
         assertTrue(data4.get(0).containsKey("id"));
-        assertTrue(data4.get(0).get("id").contains(url4));
-        assertTrue(data4.get(0).get("label").contains("Object 4"));
-        assertTrue(data4.get(0).get("type").contains("http://fedora.info/definitions/v4/repository#Container"));
+        assertTrue(data4.get(0).get("type").contains("http://fedora.info/definitions/v4/repository#Binary"));
         assertTrue(data4.get(0).get("type").contains("http://fedora.info/definitions/v4/repository#Resource"));
-        assertTrue(data4.get(0).get("type").contains("http://pcdm.org/models#Object"));
         assertTrue(data4.get(0).get("hasParent").contains(url2));
```

Comment:
The `data4` resource is a NonRDFSource -- the only triples on the RDFSource that describes the LDP-NR will be server managed (e.g. `rdf:type` and some `fedora:*` triples)